project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/js/main.js

```
@@ -0,0 +1,218 @@
+   * Initiate glightbox
+   */
+  const glightbox = GLightbox({
+    selector: '.glightbox'
+  });
+
+  /**
+   * Init swiper sliders
+   */
+  function initSwiper() {
+    document.querySelectorAll(".init-swiper").forEach(function(swiperElement) {
+      let config = JSON.parse(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)